friends/src/App.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+import FriendList from './components/FriendList';
+import FriendForm from "./components/FriendForm";
+
+
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      friends: [{
+        id: '',
+        name: '',
+        age: '',
+        email: ''
+      }],
+        name: '',
+        age: '',
+        email: ''
+    };
+
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(response => {
+        this.setState({friends: response.data});
+      })
+      .catch(error => console.log('ERROR: ', error));
+  }
+
+
+  addNewFriend = ev => {
+    ev.preventDefault();
+    if(this.state.name) {
+      axios.post('http://localhost:5000/friends', {
+        name: this.state.name,
+        age: this.state.age,
+        email: this.state.email
+      })
+      .then(response => {
+        this.setState({friends: response.data, name: '', age: '', email: ''})
+      })
+      .catch(error => console.log('ERROR: ', error));
+    }
+  };
+
+  handleInput = ev => {
+    this.setState({[ev.target.name]: ev.target.value})
+  };
+
+
+   deleteFriend = (id) => {
+     axios.delete(`http://localhost:5000/friends/{$id}`)
```

Comment:
On line 58 for fixing your delete handler you want to make sure that in the template literal you do `${id}` instead of `{$id}`